yesod-test/Yesod/Test.hs

```diff
@@ -591,12 +605,57 @@ nameFromLabel label = do
 -- > <form method="POST">
 -- >   <label>Username <input name="f1"> </label>
 -- > </form>
+--
+-- Note that this function finds the labels in which contain the given word.
+-- It might choice labels unexpectedly or just fail in the circumstances like below:
```

Comment:
Can you make this more explicit about how the function would fail? Something like the following:

> Warning: This function looks for any label that contains a subset of the provided text. If multiple labels contain that subset, this function will throw an error, as in the example below:

Site note since it sounds like you're working on your English skills:

"labels in which contain" should be "labels which contain"

"It might choice labels" should be "It might choose labels"

Reply:
> Can you make this more explicit about how the function would fail?

Sure.

And thank you for your advice about my English. It helps me a lot.